src/twisted/names/test/test_dns.py

```diff
@@ -442,6 +447,23 @@ def _recordRoundtripTest(self, record):
         self.assertEqual(record, replica)
 
 
+    def _recordEncodingTest(self, record, expectedEncoding):
```

Comment:
I know that the previous function was based on this convention, but for new code I think that  we could do better.

This is testing code, so unless there is some method which should be used by a single test, we can mark any method used by more tests as public.  `_recordEncodingTest` vs `recordEncodingTest`

In the testing code, we should reserve the private things only for code which should not be touched :)

---------

To make the tests easier to round, I would like to see domain specific language assertions.
So instead of `_recordEncodingTest` this should be something like `assertWireFormat` or something like that.

This test case is all about records... so no need for `record` in the function name... and `encoding` is very generic.

And the whole code from here is about tests, so no need for the `Test` suffix.

```
self._recordEncodingTest(rr, b'\x02\x01' + fp)
vs
self.assertWireFormat(b'\x02\x01' + fp, rr)
```


Minor comment: To make the tests easier to read, as a convention I would like to have new methods writtern as `assert(expected, actual)`

Reply:
That makes sense, I'll rename the method.

I think `encoding` is the right word here, though, because it's testing the `encode()` method of the `IEncodable` protocol.

Maybe `assertEncodedFormat` or `assertEncodedBytes` ?